### ISSUE TITLE
Have Add Const block automatically up-size a scalar constant to a vector constant

### ISSUE THREAD

**Issue by dkozel** [gnuradio/gnuradio#5974]
### Feature Description

When a vector length is specified in the Add Const block then the Constant value must be a vector with a matching length. We could easily create a vector from a scalar constant to improve the user experience of the block. This likely applies to other basic math operation blocks.

### Feature Urgency

low (just an idea)

**Comment by willcode** [comment]
Agree, and the semantics are then like numpy. This could be done in the yml files for both Python and C++, to avoid the need for additional constructors.

As an aside, it's unfortunate that multiplying an array/list by a scalar means different things in numpy and plain old Python!
